docs/content/preview/migrate/migrate-steps.md

```diff
@@ -308,7 +308,14 @@ yb-voyager import data file --export-dir <EXPORT_DIR> \
         –-has-header
 ```
 
-Refer to [import data file](../reference/yb-voyager-cli/#import-data-file) for details about the arguments.
+Import data file also allows you to load directly from your data files stored on AWS S3. The S3 bucket URI can be provided to the `data-dir` flag as follows:
```

Comment:
Can we have a separate subsection under import data for import from S3? Similar to how we have a separate section for import data from file

Reply:
addressed it. PTAL @rahulb-yb